.travis.yml

```diff
@@ -1,58 +1,99 @@
 language: php
 
-sudo: required
-dist: trusty
-group: edge
-
 php:
   - 5.5
   - 5.6
-  - hhvm
+  - 7.0
+  - 7.1
+  - 7.2
 
 env:
-  - TYPO3_BRANCH=master COVERAGE=0
-  - TYPO3_BRANCH=TYPO3_6-2 COVERAGE=0
+  - TYPO3_VERSION=^6.2 COVERAGE=0
+  - TYPO3_VERSION=^7.6 COVERAGE=0
+  - TYPO3_VERSION=^8.7 COVERAGE=0
+  - TYPO3_VERSION=dev-master COVERAGE=0
 
 matrix:
   exclude:
-    - env: TYPO3_BRANCH=master COVERAGE=0
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
+      php: 7.2
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
+      php: 7.1
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
+      php: 7.0
+    - env: TYPO3_VERSION=^8.7 COVERAGE=0
+      php: 5.5
+    - env: TYPO3_VERSION=^8.7 COVERAGE=0
+      php: 5.6
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
       php: 5.5
-    - env: TYPO3_BRANCH=master COVERAGE=0
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
       php: 5.6
-    - env: TYPO3_BRANCH=master COVERAGE=0
-      php: hhvm
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
+      php: 7.0
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
+      php: 7.1
   include:
-    - php: 5.5
-      env: TYPO3_BRANCH=TYPO3_6-2 COVERAGE=1
+    - env: TYPO3_VERSION=^7.6 COVERAGE=1
+      php: 5.6
+  allow_failures:
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
```

Comment:
Why do you allow failure on 6.2? we want to include them until they fails, and when they fail we will delete the compatibility.